2021-12-16 20:11:49.753234 | API is disabled on startup
2021-12-16 20:11:49.754235 | Settings loaded!
2021-12-16 20:11:50.552977 | Started repair process...
2021-12-16 20:11:50.553978 | Propting user to select VRCA...
2021-12-16 20:11:52.843740 | VRCA selected: C:/Users/adamc/Downloads/Avatar-Pandas-Demon-Compendium-.file_e3f724e6-61fd-4aec-8ec5-a0e47e5e27c7.2_1.vrca
2021-12-16 20:11:52.844742 | Starting repair on new thread...
2021-12-16 20:12:26.722173 | An error occured during repair: Traceback (most recent call last):
  File "C:\Users\adamc\Documents\GitHub\A.R.E.S\ARES(rewritten)\ExternalFunctions.py", line 224, in RepairVRCA
    EventLog("Thread closed and repair complete!")
TypeError: EventLog() missing 1 required positional argument: 'Data'

